Dockerfile

```diff
@@ -38,6 +38,9 @@ RUN GOARCH=$(echo "$TARGETPLATFORM" | cut -d'/' -f2) go build \
    -ldflags="-X 'github.com/getAlby/hub/version.Tag=$TAG'" \
    -o main cmd/http/main.go
 
+RUN GOARCH=$(echo "$TARGETPLATFORM" | cut -d'/' -f2) go build \
+   -o migrate cmd/db_migrate/main.go
```

Comment:
minor: could you change the output name that it's explicitly for db migration (e.g. `migrate_db`)